bluzelle/bluzelle-js
# KEP-1374/KEP-1373/KEP-1371

- Test rig up and running like before, except now it's going through Ethereum and using the secret Bluzelle creation key.
- Steps to test: 
    1. Call `webpack` to ensure protobuf files are the latest.
    2. Have Ganache running on port 8545
    3. Run `scripts/run-swarms.rb a b c ...` where a, b, c, ... the number of nodes in each swarm. For instance `run-swarms.rb 3 4 5` creates three swarms with three, four, and five nodes respectively.
    4. Copy the contract address in the output of `run-swarms.rb` and paste it into `contract_address` in `src/test/main.test.js`.
    5. Run the tests via `npm run test-node` or `npm run test-browser`.
- This PR includes a passing `expire()` test. (KEP-1291)
